src/utils/hooks/api/xcm/use-xcm-bridge.ts

```diff
@@ -119,13 +119,20 @@ const useXCMBridge = (): UseXCMBridge => {
 
           const amount = newMonetaryAmount(transferableBalance, (currency as unknown) as CurrencyExt, true);
           const balanceUSD = convertMonetaryAmountToValueInUSD(amount, getTokenPrice(prices, token)?.usd);
-          const originFee = atomicToBaseAmount(inputConfig.estimateFee, nativeToken as CurrencyExt);
+          const originFeeToAtomic = atomicToBaseAmount(inputConfig.estimateFee, nativeToken as CurrencyExt);
+
+          // TODO: This is a workaround until this is implemented on the bridge. USDT is supported as a
```

Comment:
@sander2 you can ignore this—the value returned here is being used for display only, not passed to the transaction